src/index.html

```diff
@@ -5,8 +5,412 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo_link" href="https://mate.academy/pl">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav_list">
+          <li class="header__nav_list-item">
+            <a href="#" class="header__nav_list-item-link is-active">apple</a>
+          </li>
+
+          <li class="header__nav_list-item">
+            <a href="#" class="header__nav_list-item-link">samsung</a>
+          </li>
+
+          <li class="header__nav_list-item">
+            <a href="#" class="header__nav_list-item-link">smartphones</a>
+          </li>
+
+          <li class="header__nav_list-item">
+            <a
+               href="#"
+               class="header__nav_list-item-link"
+               data-qa="nav-hover"
+            >
+               laptops & computers
+            </a>
+          </li>
+
+          <li class="header__nav_list-item">
+            <a href="#" class="header__nav_list-item-link">gadgets</a>
+          </li>
+
+          <li class="header__nav_list-item">
+            <a href="#" class="header__nav_list-item-link">tablets</a>
+          </li>
+
+          <li class="header__nav_list-item">
+            <a href="#" class="header__nav_list-item-link">photo</a>
+          </li>
+
+          <li class="header__nav_list-item">
+            <a href="#" class="header__nav_list-item-link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+
+    <main class="catalog">
+      <div class="catalog__cart" data-qa="card">
```

Comment:
```suggestion
      <article class="catalog__cart" data-qa="card">
```
Consider using semantic tags